.travis.yml

```diff
@@ -2,21 +2,19 @@ language: php
 
 matrix:
   include:
-  - php: 7
+  - php: 7.1
```

Comment:
I switched composer.json to ~7.1, so can't list 7.0 here. Doing this in dedicated PR so I can more easily use 7.1 features afterwards without complicating PRs about other things.